framework/src/modules/pos/types.ts

```diff
@@ -308,3 +308,7 @@ export interface ClaimableReward {
 	tokenID: string;
 	reward: string;
 }
+
+export interface RegistrationFeeResponse {
+	fee: string;
+}
```

Comment:
This is a very tiny remark, but that's because everything else already looks good 🙂

I think I personally wouldn't define a new type for something as simple as `{ fee: string }`, to avoid fatigue from having too many types. The same way you you did not define a new type for the simple responses of `hasUserAccount()` and `hasEscrowAccount()` and used object literal `{ exists: boolean }` instead.

But since you had defined it, then let's make it consistent with the other endpoint response types 😉

Looking at the other definitions of request/response types here and in other files, it looks like the convention is to include the full name of the endpoint, with `Request` or `Response` as suffix.

Therefore, `GetRegistrationFeeResponse` would be the consistent name here. The same goes for the definition in Interoperability module.